src/fetch/orderbook.py

```diff
@@ -0,0 +1,251 @@
+"""Basic client for connecting to postgres database with login credentials"""
```

Comment:
If we really want to merge the two codebases, this needs to be combined with `pg_client.py`.